test/ibmq/test_ibmq_job_attributes.py

```diff
@@ -263,8 +295,21 @@ def test_queue_info(self, provider):
             self.assertTrue(all(0 < priority <= 1.0 for priority in [
                 queue_info.hub_priority, queue_info.group_priority, queue_info.project_priority]),
                             "Unexpected queue info {} for job {}".format(queue_info, job.job_id()))
-            self.assertTrue(queue_info.format())
-            self.assertTrue(repr(queue_info))
+
+            queue_info_format = queue_info.format()
+            self.assertTrue(queue_info_format)
+            queue_info_repr = repr(queue_info)
+            self.assertTrue(queue_info_repr)
+
+            # Convert the utc start and end times to local time, represented as a string.
+            est_start_time_local_str = datetime_to_str(
+                utc_to_local(queue_info.estimated_start_time))
+            est_complete_time_local_str = datetime_to_str(
+                utc_to_local(queue_info.estimated_complete_time))
+
+            # Ensure the local start and end date times are in the `repr` format.
+            self.assertIn(est_start_time_local_str, queue_info_repr)
```

Comment:
What's the point of the doing the assert if you're just calling the same exact functions as the mainline code?